tests/integration/db/Questionnaire.test.js

```diff
@@ -0,0 +1,117 @@
+process.env.NODE_ENV = "test";
+
+const knex = require("../../../db");
+
+const migration = require("../../../migrations/20170620163533_create_questionnaires_table");
```

Comment:
Why do you need to manually run the migration?

Reply:
For this integration test I needed the questionnaire table in place. I originally ran all migrations on a beforeAll step (against a SQLite database). Unfortunately, some of the migrations use features available in postgres, but not in SQLite. So running all migrations is not possible. For the purposes of the test. All I care about is the existence of the table. So I'm able to run that migration in isolation.